libsrc/meshing/basegeom.hpp

```diff
@@ -339,15 +339,12 @@ namespace netgen
   public:
     virtual ~GeometryRegisterArray()
```

Comment:
Avoid accidental misuse (copying) of the singleton.
Make non-virtual, singleton should never be reimplemented.
```suggestion
  public:
    GeometryRegisterArray(const GeometryRegisterArray &) = delete;
    GeometryRegisterArray & operator=(const GeometryRegisterArray &) = delete;
  private:
    GeometryRegisterArray() = default;
    ~GeometryRegisterArray() = default;
```